docs/uncertainty/index.rst

```diff
@@ -0,0 +1,353 @@
+.. |quantity| replace:: :class:`~astropy.units.Quantity`
+.. |distribution| replace:: :class:`~astropy.uncertainty.Distribution`
+
+.. _astropy-uncertainty:
+
+*******************************************************
+Uncertainties and Distributions (`astropy.uncertainty`)
+*******************************************************
+
+.. warning::
+
+    `astropy.uncertainty` is currently a work-in-progress, and thus it is quite
+    possible there will be API changes in later versions of Astropy. If you have
+    specific ideas for how it might be improved, please  let us know on the
+    `astropy-dev mailing list`_ or at http://feedback.astropy.org .
+
+Introduction
+============
+
+In addition to |quantity|, astropy provides a |distribution| object to represent
+statistical distributions in a form that acts as a drop-in replacement for
+|quantity| or regulay Numpy arrays. Used in this manner, |distribution| provides
+uncertainty propogation at the cost of additional computation.  It can also
+more generally represent sampled distributions for e.g., Monte Carlo calculation
+techniques.
+
+The core object for this feature is the |distribution|.  Currently, all
+such distributions are Monte Carlo sampled.  While this means each distribution
+may take more memory, it allows arbitrarily complex operations to be performed
+on distributions while maintaining their correlation structure. Some specific
+well-behaved distributions (e.g., the Normal distribution) have
+analytic forms which may eventually enable a more compact/efficient
+representation, but this is not currently implemented.

```

Comment:
We might want to at least mention how this relates to the uncertainty stuff in the nddata package, and point out that we might somehow combine these in future?

Reply:
I added a few words that amount to "this might happen"